### PR TITLE
feature(commo):add view Menu Sidebar

### PR DESCRIPTION
**Pull request by leonel-07**
Sidebar menu with mobile and desktop functionality

### CODE REVIEW

File: src/ui/components/menuStorybook/BurgerButton.js [hunk 1]
```diff
@@ -0,0 +1,13 @@
+import React, { useState } from 'react';
+const BurgerButton = (props) => {
+    return (
+        <div className = "burger">
+            <div onClick = { ()=>{props.handlerCLick();}} className = "blocksButton">
+                <div/>
```
Comment (Ditmar) [inline src/ui/components/menuStorybook/BurgerButton.js:6]: the markup is not closed as it should be, must be close as </div>

File: src/ui/components/menuStorybook/BurgerButton.js [hunk 2]
```diff
@@ -0,0 +1,13 @@
...
+                <div/>
```
Comment (Ditmar) [inline src/ui/components/menuStorybook/BurgerButton.js:7]: the markup is not closed as it should be, must be close as </div>

File: src/ui/components/menuStorybook/BurgerButton.js [hunk 3]
```diff
@@ -0,0 +1,13 @@
...
+                <div/>
```
Comment (Ditmar) [inline src/ui/components/menuStorybook/BurgerButton.js:8]: the markup is not closed as it should be, Must be close as </div>

**Review by Ditmar — COMMENTED** [review]
Great job only littel commets

**Comment by olmedo90** [comment]
Captura de pantalla (774)

Es ACORDE al figma propuesto.
Cuenta con STORYBOOK.
like

File: src/ui/components/menuStorybook/BurgerButton.js (hunk 3 above)
Comment (leonel-07) [reply src/ui/components/menuStorybook/BurgerButton.js:7]: resolve

File: src/ui/components/menuStorybook/Sidebar_menu.js [hunk 4]
```diff
@@ -0,0 +1,68 @@
+import React, { useState } from "react";
```
Comment (Ditmar) [inline src/ui/components/menuStorybook/Sidebar_menu.js:1]: Please use single quote

File: src/ui/components/menuStorybook/Sidebar_menu.js [hunk 5]
```diff
@@ -0,0 +1,68 @@
...
+import BurgerButton from "./BurgerButton";
```
Comment (Ditmar) [inline src/ui/components/menuStorybook/Sidebar_menu.js:2]: Please use single quote

File: src/ui/components/menuStorybook/Sidebar_menu.js [hunk 6]
```diff
@@ -0,0 +1,68 @@
...
+import { BrowserRouter as Router, Switch, Route, Link } from "react-router-dom";
```
Comment (Ditmar) [inline src/ui/components/menuStorybook/Sidebar_menu.js:3]: Please use single quote

File: src/ui/components/menuStorybook/Sidebar_menu.js [hunk 7]
```diff
@@ -0,0 +1,68 @@
...
+import icon_11 from "./iconos/icon_11.png";
```
Comment (Ditmar) [inline src/ui/components/menuStorybook/Sidebar_menu.js:4]: Please use single quote

File: src/ui/components/menuStorybook/Sidebar_menu.js [hunk 8]
```diff
@@ -0,0 +1,68 @@
...
+import icon_22 from "./iconos/icon_22.png";
```
Comment (Ditmar) [inline src/ui/components/menuStorybook/Sidebar_menu.js:5]: Please use single quote

File: src/ui/components/menuStorybook/Sidebar_menu.js [hunk 9]
```diff
@@ -0,0 +1,68 @@
...
+import icon_33 from "./iconos/icon_33.png";
```
Comment (Ditmar) [inline src/ui/components/menuStorybook/Sidebar_menu.js:6]: Please use single quote

File: src/ui/components/menuStorybook/Sidebar_menu.js [hunk 10]
```diff
@@ -0,0 +1,68 @@
...
+import icon_44 from "./iconos/icon_44.png";
```
Comment (Ditmar) [inline src/ui/components/menuStorybook/Sidebar_menu.js:7]: Please use single quote

File: src/ui/components/menuStorybook/Sidebar_menu.js [hunk 11]
```diff
@@ -0,0 +1,68 @@
...
+import icon_55 from "./iconos/icon_55.png";
```
Comment (Ditmar) [inline src/ui/components/menuStorybook/Sidebar_menu.js:8]: Please use single quote

File: src/ui/components/menuStorybook/Sidebar_menu.js [hunk 12]
```diff
@@ -0,0 +1,68 @@
...
+import icon_66 from "./iconos/icon_66.png";
```
Comment (Ditmar) [inline src/ui/components/menuStorybook/Sidebar_menu.js:9]: Please use single quote

File: src/ui/components/menuStorybook/Sidebar_menu.js [hunk 13]
```diff
@@ -0,0 +1,68 @@
...
+import icon_77 from "./iconos/icon_77.png";
+import logo1 from "./iconos/logo1.png";
```
Comment (Ditmar) [inline src/ui/components/menuStorybook/Sidebar_menu.js:11]: Please use single quote

File: src/ui/components/menuStorybook/Sidebar_menu.js [hunk 14]
```diff
@@ -0,0 +1,68 @@
...
+import "./Sidebar_menu.scss";
```
Comment (Ditmar) [inline src/ui/components/menuStorybook/Sidebar_menu.js:12]: Please use single quote

**Review by Ditmar — COMMENTED** [review]
good catch but change the double quotes by single quotes

File: src/ui/components/menuStorybook/Sidebar_menu.js (hunk 14 above)
Comment (leonel-07) [reply src/ui/components/menuStorybook/Sidebar_menu.js]: changes made

File: src/ui/components/menuStorybook/Sidebar_menu.scss [hunk 15]
```diff
@@ -0,0 +1,222 @@
...
+            font-style: normal;
+            font-weight: 700;
+            font-size: 1.563rem;
+            line-height: 1.5rem;
+            letter-spacing: 0.025rem;
+            color: #A4A6B3;
+            opacity: 0.7;
+        }
+    }
+    .logoLeonel{
```
Comment (Ditmar) [inline src/ui/components/menuStorybook/Sidebar_menu.scss:156]: please change this name

File: src/ui/components/menuStorybook/Sidebar_menu.js [hunk 16]
```diff
@@ -0,0 +1,68 @@
...
+        setCliked( !Clicked );
+    }
+    return (
+        <div>
+            <div className="burger">
+            <BurgerButton Clicked = { Clicked } handlerCLick = { handlerCLick }/>
+            </div>
+            <div className={Clicked ? 'sidebarMovil':'sidebarDesktop'}>
+                <ul>
+                    <div className="box"><img className="logoLeonel" src={ logo1 }/>
```
Comment (Ditmar) [inline src/ui/components/menuStorybook/Sidebar_menu.js:27]: change this name

**Review by Ditmar — COMMENTED** [review]
plese change the name of the class logoLeonel for another

File: src/ui/components/menuStorybook/Sidebar_menu.js (hunk 16 above)
Comment (leonel-07) [reply src/ui/components/menuStorybook/Sidebar_menu.js]: Cambiado inge

**Review by Ditmar — APPROVED** [review]
nice pr

**Pull request merged**